lib/parallel_tests/test/runner.rb

```diff
@@ -85,6 +85,39 @@ def tests_with_size(tests, options)
           tests
         end
 
+        def process_in_batches(cmd, os_cmd_length_limit, tests)
+          # Filter elements not starting with value in tests to retain in each batch
+          # i.e. retain common parameters for each batch
+          retained_elements = cmd.reject { |s| s.start_with?(tests) }
```

Comment:
isn't tests files[:first] so just a file and not a prefix ?

Reply:
when we start the process we give features/ and files[:first] has that .. which has been expanded into individual files into cmd array

Reply:
ahh ... yeah that's a bit brittle, user could be giving multiple folders too ... I'll see if I can find a better way, otherwise keeping with a comment

Reply:
won't work with minitest since that uses requires and not a list of files
`"%w[#{require_list}].each { |f| require %{./\#{f}} }",`